tests/unit/test_utils_windows_executable.py

```diff
@@ -14,6 +14,7 @@
 
 
 @pytest.mark.utils
+@pytest.mark.skip_windows
```

Comment:
Why are we skipping windows tests on windows, what is the back story here?

Reply:
Ah. Good catch, I probably meant this:

```
@pytest.mark.skipif(os.name != "nt", reason="Windows file paths tested")
```